install-kubeadm.bash

```diff
@@ -12,9 +12,31 @@ apt-get update
 
 apt-get install -y docker-ce=17.03.2~ce-0~ubuntu-xenial
 
+COMMON_DOCKER_PROPS='"storage-driver": "overlay2"'
+
+# See if a docker volume device was provided
+if [ "$#" = 1 ]; then
+  mkfs -t ext3 $1
+  mkdir /opt/docker
+  mount $1 /opt/docker
+  DOCKER_VOL_PROPS='"graph":"/opt/docker"'
```

Comment:
I'd add a comment here, saying that in the future we will want to switch to `data-root`